src/brain/issueLabelHandler/followups.ts

```diff
@@ -70,37 +68,65 @@ export async function updateCommunityFollowups({
   const repo = payload.repository.name;
   const issueNumber = payload.issue.number;
 
-  const isWaitingForCommunityLabelOnIssue = payload.issue.labels?.find(
-    ({ name }) => name === WAITING_FOR_COMMUNITY_LABEL
-  )?.name;
+  const isLabelOnIssue = (labelName) =>
+    payload.issue.labels?.find(({ name }) => name === labelName)?.name;
 
-  if (isWaitingForCommunityLabelOnIssue) {
-    await org.api.issues.removeLabel({
+  if (
+    (await isNotFromAnExternalOrGTMUser(payload)) ||
```

Comment:
nit: its a bit confusing that one of these is `isNot...`, and the other `is...`. Took me a while to figure out what's what, would probably be a bit easier to read it we had `!isFromAnExternalOrGTMUser(payload) ||| isCommentFromCollaborator(payload)`.

Aside: what is a `GTMUser`?

Reply:
`GTMUser` is anyone from go to market! So that should cover sales and support

Reply:
Going to not go with this change, as it is an imported function and is used in many places

Reply:
Sgtm.